bootstrapvz/plugins/minimize_size/__init__.py

```diff
@@ -54,8 +54,8 @@ def resolve_tasks(taskset, manifest):
         if apt.get('autoremove_suggests', False):
             taskset.add(tasks.apt.AptAutoremoveSuggests)
     if 'dpkg' in manifest.plugins['minimize_size']:
-        filter_tasks = [tasks.dpkg.CreateDpkgCfg,
-                        tasks.dpkg.InitializeBootstrapFilterList,
+        taskset.add(tasks.dpkg.CreateDpkgCfg)
```

Comment:
What's the motivation for this change?

Reply:
The path for the dpkg config needs to be created so the setting `force-unsafe-io`, so I wanted to make sure the task CreateDpkgCfg was called regardless of is set in the manifest file for` minimize_size: { dpkg: {} }`. But that still seems hacky. So I have change to just creating the dir in the Docker provider.